nowcasting_utils/models/losses/FocalLoss.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import torch
+from torch import nn as nn
+
+
+class FocalLoss(nn.Module):
+    def __init__(
+        self,
+        apply_nonlin=None,
+        alpha=None,
+        gamma=2,
+        balance_index=0,
+        smooth=1e-5,
+        size_average=True,
+    ):
+        """
+        Copied from: https://github.com/Hsuxu/Loss_ToolBox-PyTorch/blob/master/FocalLoss/FocalLoss.py
```

Comment:
github.com/Hsuxu/Loss_ToolBox-PyTorch doesn't have an explicit license... Should we ask hsuxu if we can copy this code?  And, if so, how hsuxu would like to be attributed?

Reply:
I've switched it out to an explicitly MIT licensed version!